pylintrc

```diff
@@ -2,7 +2,6 @@
 
 disable=
   missing-docstring,
-  bad-continuation,
```

Comment:
Why did you remove that?

Reply:
Because the pylint would "warn" otherwise that this is deprecated.

Reply:
```
************* Module pylintrc
pylintrc:1:0: R0022: Useless option value for '--disable', 'bad-continuation' was removed from pylint, see https://github.com/pylint-dev/pylint/pull/3571. (useless-option-value)
```

Reply:
Yes, I know. Unfortunately current pylint on current openSUSE Leap 15.5 still has the check included and would actually fail style checks if not ignored. We could only exclude "bad-continuation" based on parsing the version of pylint itself on invocation

Reply:
But that's not the reason for failure, that is

```
************* Module openqabot.types.incident
openqabot/types/incident.py:190:1: W0511: TODO: (fixme)
```

Reply:
Both need to be removed iirc. let me remove the commit and see

************* Module pylintrc
pylintrc:1:0: R0022: Useless option value for '--disable', 'bad-continuation' was removed from pylint, see https://github.com/pylint-dev/pylint/pull/3571. (useless-option-value)
************* Module openqabot.types.incident
openqabot/types/incident.py:190:1: W0511: TODO: (fixme)

Reply:
@okurz removing that line, the warning still shows up: https://github.com/openSUSE/qem-bot/actions/runs/7757445750/job/21156903452#step:6:17

I leave it to you for a different PR if you want to remove such warning, I doubt Leap version has anything to do with linting rules.